### PR TITLE
Update opflex docker file to point to newer base image

### PR DESCRIPTION
**Pull request by vlella**

### CODE REVIEW

**Comment by snaiksat** [comment]
Change looks fine to me, we would just need to properly regression test this change.

File: docker/Dockerfile-opflex [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ARG basetag=latest
 ARG baserepo=quay.io/noirolabs
-FROM ${baserepo}/aci-containers-base:${basetag}
-RUN yum install -y --enablerepo=openstack-15-for-rhel-8-x86_64-rpms \
+FROM ${baserepo}/aci-containers-base:kmr2-test
```
Comment (snaiksat) [inline docker/Dockerfile-opflex:3]: We should not be changing this, it will automatically get set to the relevant base image from the Jenkins jobs.

Comment (vlella) [reply docker/Dockerfile-opflex:3]: yes, I will revert this back later but for current testing purpose to build an image via jenkins job using this branch had to tweak this. 

Image -> https://noiro-quay.cisco.com/repository/noiro/opflex?tab=tags health-index-test

job used with names -> aci-containers-opflex and  aci-containers-opflex-build-base in  https://engci-jenkins-sjc.cisco.com/jenkins/job/team_noiro_engineering/view/Users/job/Vishal/ 

@snaiksat

**Comment by snaiksat** [comment]
Closing this since (1) the base image is not correct (2) the vulnerabilities issue which this PR was attempting to solve was fixed by picking up the correct base image in the build process (it was not related to the OpenStack package which is being changed here).

**Pull request closed without merging**